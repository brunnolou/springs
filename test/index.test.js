import springs from '../src';

test('should run without errors', () => {
  springs(30, 7);
});

it('Should return a different value after some time', (done) => {
  const s1 = springs(30, 0.1);

  s1(1000);

  setInterval(() => {
    expect(s1(0)).toMatchSnapshot();

    done();
  }, 30);
});
