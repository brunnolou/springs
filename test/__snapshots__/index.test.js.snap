// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should show the logged in user name after successful login 1`] = `0.08080103108041667`;
